domino_generator/example/ex1/ex1.g.dart

```diff
@@ -2,14 +2,16 @@ import 'package:domino/src/experimental/idom.dart' as _i0 show DomContext;
 import 'package:meta/meta.dart' as _i2 show required;
 import 'ex1_model.dart' as _i1 show Example;
 
-void renderEx1(
+void Ex1(
```

Comment:
Let's not remove the `render` prefix! I think it is a nice convention to add if the component name starts with an uppercase. (in the `_dartName` function - if I read the change correctly).